### DISCUSSION TITLE
view and annotate external slides

### DISCUSSION THREAD

**Discussion by mdrio** [DigitalSlideArchive/digital_slide_archive#294] [Q&A] [answered]
Hi , DSA looks a very interesting project. I wonder if there is a way to view and annotate external slides. At the moment I have slides on an OMERO server, and I visualize them using a plugin that works as a DZI server on top of OMERO (https://github.com/crs4/ome_seadragon). Is it possible to connect DSA to OMERO or to view remote dzi files?
Thanks

**Comment by manthey** [comment]
There isn't currently a direct way to do this, but it is possible.

DSA is built on top of Girder, which has the concept of assetstores.  In general, files can be uploaded (transferred to Girder) or imported (referenced where they are so no bytes are moved). It is also possible to have federated data, where you browse remote files and then can access them.  In this case, the annotations would still need to be stored in the DSA database.

**Reply by mdrio** [reply]
That's interesting. I couldn't find documentation about referencing data on an assetstore or about the federated solution, can you provide some links or examples?

**Reply by manthey** [reply, marked as answer]
The core Girder docs reference using different assetstores (e.g., https://girder.readthedocs.io/en/latest/user-guide.html#s3 for s3).  We also have just implemented using DICOMweb as an assetstore (but it is new enough there isn't any docs on that yet other than the commentary in the issues).

For federated solutions, those have so far been custom per specific clients; I'm not sure we have a recent example that I can point to.
